posts/2020/11/Design In Open Source

```diff
@@ -0,0 +1,187 @@
+ <!--
+.. title: Introduction to Design in Open-Source
+.. slug: introduction-to-design-in-open-source
+.. date: 2020-11-13 00:07:30 UTC-05:00
```

Comment:
This post will only be rendered in CI if the publication date is in the past. Could you update it to yesterday?

Reply:
OK, I think I fixed that.